### ISSUE TITLE
Joyride requires hash fragment?

### ISSUE THREAD

**Issue by saibotsivad** [zurb/joyride#204]
I was attempting to use Joyride inside a web app, to show off new features, but like most web apps ours uses the hash fragment for navigation. E.g. `site.com/#/company/123` to show the company page for the company id "1234".

After successfully got Joyride running, I now see that all the clickable actions (e.g. "next", "close", etc.) are actually `<a>` elements, with the `href` attribute containing a hash fragment. E.g. `<a href="#close">X</a>` for the close action.

Is there any way to override this? It looks pretty integral to Joyride, but using hash fragments effectively rules out using Joyride for any typical web app, since nearly all of them use the hash fragment for application routing.

**Comment by saibotsivad** [comment]
I'm not sure if there is a better way, but I resolved this in AngularJS by doing this:

```js
var element = $('#' + slide.id);
element.joyride(options);
element.joyride(options);
$timeout(function() {
	$('.joyride-tip-guide a[href]').each(function() {
		$(this).click(function(event) {
			event.preventDefault();
		})
	})
})
```

(Running joyride twice because of Issue 200)

**Comment by robinfhu** [comment]
Yes you can override this.  You should be able to change the outer HTML of the close button and next button in the settings:
```
$('#tutorial').joyride({
    template: {
        'link': '<a href="javascript:void(0);" class="joyride-close-tip">X</a>
    }

})
```
